storybookjs/storybook
# Fixed react peer dependency

## What I did

Changed `|` to `||` to avoid warnings when installing `@storybook/react`

## How to test

Run `npm install @storybook/react` and this warning should be gone:

> npm WARN @storybook/react@3.3.1 requires a peer of babel-core@^6.26.0 | ^7.0.0-0 but none was installed.
Is this testable with jest or storyshots?

Does this need a new example in the kitchen sink apps?

No.

Does this need an update to the documentation?

No.
